[debug] [2024-09-09T22:06:12.603Z] ----------------------------------------------------------------------
[debug] [2024-09-09T22:06:12.606Z] Command:       /home/dyloge/.nvm/versions/node/v18.15.0/bin/node /home/dyloge/.nvm/versions/node/v18.15.0/bin/firebase login
[debug] [2024-09-09T22:06:12.606Z] CLI Version:   13.16.0
[debug] [2024-09-09T22:06:12.607Z] Platform:      linux
[debug] [2024-09-09T22:06:12.607Z] Node Version:  v18.15.0
[debug] [2024-09-09T22:06:12.607Z] Time:          Tue Sep 10 2024 01:36:12 GMT+0330 (Iran Standard Time)
[debug] [2024-09-09T22:06:12.608Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-09-09T22:06:12.611Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[info] i  Firebase optionally collects CLI and Emulator Suite usage and error reporting information to help improve our products. Data is collected in accordance with Google's privacy policy (https://policies.google.com/privacy) and is not used to identify you.
 
[info] i  To change your data collection preference at any time, run `firebase logout` and log in again. 
[info] 
[info] Visit this URL on this device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=294763467&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2024-09-09T22:06:28.314Z] >>> [apiv2][query] POST https://accounts.google.com/o/oauth2/token [none]
[debug] [2024-09-09T22:06:28.315Z] >>> [apiv2][body] POST https://accounts.google.com/o/oauth2/token [omitted]
[debug] [2024-09-09T22:08:26.678Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: connect ETIMEDOUT 34.120.206.254:443
[warn] âš   Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2024-09-09T22:08:26.679Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[debug] [2024-09-09T22:08:43.058Z] *** [apiv2] error from fetch(https://accounts.google.com/o/oauth2/token, {"headers":{},"method":"POST","body":{"_overheadLength":543,"_valueLength":208,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------196197160549286977331773","_events":{},"_eventsCount":1}}): FetchError: request to https://accounts.google.com/o/oauth2/token failed, reason: connect ETIMEDOUT 64.233.167.84:443
[debug] [2024-09-09T22:08:43.060Z] Token Fetch Error: FirebaseError: Failed to make request to https://accounts.google.com/o/oauth2/token
    at RetryOperation._fn (/home/dyloge/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/lib/apiv2.js:259:31)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[debug] [2024-09-09T22:08:43.081Z] >>> [apiv2][query] POST https://auth.firebase.tools/attest [none]
[debug] [2024-09-09T22:08:43.082Z] >>> [apiv2][body] POST https://auth.firebase.tools/attest {"session_id":"d137e034-5bf7-432e-96c7-179cca7b92ac"}
